### ISSUE TITLE
Scan2Cad training data not available

### ISSUE THREAD

**Issue by YuePanEdward** [xheon/JointEmbedding#3]
Thanks for your great work. 
I have some questions about the training data preparation.
I followed the [Scan2CAD](https://github.com/skanti/Scan2CAD) repository and download its annotation dataset. 
However, it seems that there's no training, validation split within it. Besides, there's also no field called 'scan2cad_objects' in the dataset's json file. This is also not included in the json file of the Scan-CAD Object Similarity dataset. Which file should be filled in the field of "--scan2cad_file"?
I wonder if you can give us some more instructions on the training data preparation. 
Thank you very much.

**Comment by xheon** [comment]
Thanks for pointing that out. The missing jsons have been added to the dropbox folder.

**Comment by star-cold** [comment]
> Thanks for pointing that out. The missing jsons have been added to the dropbox folder.

So where is the dropbox folder?

**Comment by YuePanEdward** [comment]
Thank you very much for the rapid reply and the provided data.    
   I am sorry that I still have a question regarding the data preparation.
   I wonder how we can generate the *.mask files from the ScanNet dataset? Since they are the necessary inputs of the network.
   Thank you in advance.

**Comment by YuePanEdward** [comment]
According to the paper, I managed to generate the training data from ScanNet, ShapeNet and Scan2CAD dataset. The problem is solved.

**Comment by hugoycj** [comment]
> According to the paper, I managed to generate the training data from ScanNet, ShapeNet and Scan2CAD dataset. The problem is solved.

Great thanks for your answer. I also met the same problem. I wonder if there is any code for generating such data
